docs/user-references.md

```diff
@@ -0,0 +1,28 @@
+# User References
+
+To show the community our users and their usage of LoopBack, we have been asking our users to reach out to us if they wish to have their companies showcased on our web site. 
+
+## Where are the references
+
+On the [LoopBack landing page](https://loopback.io/) under "Who's using LoopBack?" section, the logos of the companies which are using LoopBack will be highlighted there. 
+
+Optionally, users can write a short paragraph about their experience with LoopBack and it will be listed in the ["What our users say" page](https://loopback.io/what-our-users-say.html).
+
+## What should I do if I'm interested to have my company showcased? 
+
+Submit a PR in [loopback.io repo](https://github.com/loopbackio/loopback.io):
+1. add a high resolution company logo under [`images/users` folder](https://github.com/loopbackio/loopback.io/tree/gh-pages/images/users). It's recommended that the image is in png format. 
+2. edit the "Who's using LoopBack?" section in [`index.html` file](https://github.com/loopbackio/loopback.io/blob/gh-pages/index.html). 
+3. Optionally, a short paragraph in [`what-our-users-say.html` file](https://github.com/loopbackio/loopback.io/blob/gh-pages/what-our-users-say.html) about the usage and experience, together with the user's name, title and company. You might want to take a look at [this page](https://loopback.io/what-our-users-say.html) for inspiration and format. 
+
+## As a maintainer, what should I do to review the PR?
+
+Besides the regular review activity (make sure the pages are displaying properly, check for typo, etc), it would be good for maintainers to do some due dilligence for verification, though it is usually hard to validate whether the highlighted products are using LoopBack.
```

Comment:
I'm wondering if we should be verifying that the GitHub is part of the company (i.e. through email or some other indicators), or if it's not necessary

Reply:
@achrinza, I'm not sure if GitHub id/repo would be a good indication for a company. Maybe it doesn't have much open source contribution as a company or not having any code in the open source world? 
We can probably go to the web site of the company and possibly social media account/pages (if exists). It might be hard to validate they are actually using LoopBack in their product though :)

Reply:
I'm happy to improve this PR by adding a few possible places we can look.